RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -168,15 +149,15 @@ void Event::addHitToEvent(std::vector<float> const& x,
   unsigned int nHits = x.size();
 
   // Initialize space on device/host for next event.
-  if (hitsInGPU == nullptr) {
-    hitsInGPU = new Hits();
-    hitsBuffers = new HitsBuffer<Device>(nModules_, nHits, devAcc, queue);
+  if (!hitsInGPU) {
+    hitsInGPU = Hits();
+    hitsBuffers = HitsBuffer<Device>(nModules_, nHits, devAcc, queue);
```

Comment:
I think
```suggestion
    hitsBuffers.emplace(nModules_, nHits, devAcc, queue);
```
would avoid the copy.